manager/integration/tests/test_statefulset.py

```diff
@@ -438,3 +439,133 @@ def test_statefulset_restore(set_random_backupstore, client, core_api, storage_c
     for pod in pod_info:
         resp = read_volume_data(core_api, pod['pod_name'])
         assert resp == pod['data']
+
+
+def test_statefulset_volume_expand(client, core_api, storage_class, statefulset): # NOQA
+    """
+    Test that expanded volume can be mounted by a StatefulSet.
+
+    1. Create a StatefulSet with VolumeClaimTemplate and Longhorn.
+    2. Wait for pods to run.
+    3. Scale down replicas to 0
+    4. Expand volume using pvc
```

Comment:
This case is for testing the update HTTP API `Expand`. You need to do expansion via `v.expand()` then check if both PVC and the volume itself get updated.